Cargo.toml

```diff
@@ -20,3 +20,4 @@ toml = "0.5.6"
 clap = { version = "4.0.15", features = ["derive", "wrap_help"] }
 shlex = "1.3.0"
 serde_json = "1.0.108"
+camino = "1.1.7"
```

Comment:
I wonder, since we don't depend on any specific feature, maybe it'd be better to use a generic `1.1` version and let `cargo` take it from there? That way if the dependencies decide to update their used versions we could avoid duplicates in the tree.